PROVIDER_CHECKLIST.md

```diff
@@ -0,0 +1,115 @@
+# Provider Checklist
+
+Crossplane manages Resources external via
+[Providers](https://crossplane.io/docs/master/concepts/providers.html).
+Providers are composed of Kubernetes [Custom Resource
+Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+and [Controllers](https://kubernetes.io/docs/concepts/architecture/controller)
+that communicate to a remote API and manages the lifecycle of a resource from
+Creation to Deletion.
+
+To build a new provider, please refer to the [Provider Development
+Guide](https://crossplane.io/docs/master/contributing/provider_development_guide.html).
+
+This document contains list of items that are usually involved in managing Open
+Source contributions to a Crossplane provider. In general your provider should
+follow the guidelines documented in the [Crossplane
+Contributing](https://github.com/crossplane/crossplane/blob/master/CONTRIBUTING.md)
+guide.
+
+## Repository
+
+It is recommended to use the
+[crossplane-contrib](https://github.com/orgs/crossplane-contrib) Github
+Organization. Your project should be named `provider-<name>`. Members of the
+Crossplane community are happy to create a repository and configure access.
+Please open an issue in the Crossplane [org](https://github.com/crossplane/org)
+repository or reach out to the Crossplane
+[#dev](https://crossplane.slack.com/archives/CEF5N8X08) channel.
```

Comment:
We might want to relax this a little. Under the new charter (https://github.com/crossplane/crossplane/pull/3045) we don't really _recommend_ folks use crossplane-contrib more than any other GitHub org. Instead we offer it as a place for providers that would like a neutral home that is under Crossplane's project governance.

Reply:
I've updated the wording. Please let me know what you think